android/app/src/main/java/com/chorestory/app/ParentHomeActivity.java

```diff
@@ -11,11 +12,14 @@
 import com.chorestory.fragment.ParentProfileFragment;
 import com.chorestory.fragment.ParentQuestsFragment;
 
+import java.util.Arrays;
+import java.util.List;
+
 public class ParentHomeActivity extends ChoreStoryActivity {
 
     private final String CLAN = "Clan";
```

Comment:
I think these should go into a separate sub-package, we don't want to copy-paste them around.